config/environment.js

```diff
@@ -84,7 +84,8 @@ module.exports = function(environment) {
     // ENV.APP.LOG_VIEW_LOOKUPS = true;
 
     ENV.API_BASE_URL = 'http://api.lvh.me:49235';
-    ENV.SERVER_TOKEN_ENDPOINT = 'http://api.lvh.me:49235/login';
+    ENV.SERVER_TOKEN_ENDPOINT = 'http://api.lvh.me:49235/token';
```

Comment:
You could make your life slightly easier by storing the sub-uri's in a constant and appending that to these env vars. That way when you need to change a route, you don't have to search for every instance of "login" or such.